### PR TITLE
Workload clone

### PR DESCRIPTION
**Pull request by jordan-rash**
Depends on #439 

This adds the ability to clone an existing workload to a different node.  

```
└─❯ go run . workload copy vrf6eWwGSciHXqBlf0lziI
Workload successfully copied! New workloadId: fuH0kmX3YQXgmkmBsPxr83
```

### CODE REVIEW

File: cmd/nex/workload.go [hunk 1]
```diff
@@ -306,6 +357,64 @@ func (b BundleWorkload) Validate() error {
 	return errs
 }
 
+type CopyWorkload struct {
+	WorkloadId   string `arg:"" description:"ID of the workload"`
+	WorkloadType string `name:"type" description:"Type of workload" default:"direct_start"`
+	StopOriginal bool   `name:"stop" description:"Stop the original workload after copying" default:"false"`
+
+	NodeId   string            `description:"Node ID of target workload. If not provided, auction is preformed"`
```
Comment (jordan-rash) [inline cmd/nex/workload.go:365]: Not yet implemented

File: api/nodecontrol/clone-workload-response.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "io.nats.nex.v2.clone_workload_request",
```
Comment (autodidaddict) [inline api/nodecontrol/clone-workload-response.json:3]: this Id should be `clone_workload_response`

File: node/internal/actors/direct_start_agent.go [hunk 3]
```diff
@@ -117,14 +121,26 @@ func (a *DirectStartAgent) Receive(ctx *goakt.ReceiveContext) {
...
 			NodeId:           a.nodeId,
-			TargetXkey:       "",
+			TargetXkey:       xkpub,
 			Version:          VERSION,
 			Tags:             map[string]string{},
 			StartedAt:        timestamppb.New(a.startedAt),
 			RunningWorkloads: runningWorkloads,
 		})
+	case *actorproto.GetRunRequest:
+		a.logger.Debug("QueryWorkloads received", slog.String("name", ctx.Self().Name()))
```
Comment (autodidaddict) [inline node/internal/actors/direct_start_agent.go:138]: this should be `GetRunRequest received`

Comment (autodidaddict) [reply node/internal/actors/direct_start_agent.go:138]: Did copilot do this? :trollface:

**Review by autodidaddict — CHANGES_REQUESTED** [review]
minor nits

File: node/internal/actors/direct_start_agent.go (hunk 3 above)
Comment (jordan-rash) [reply node/internal/actors/direct_start_agent.go]: I will always blame copilot

**Pull request merged**